### PR TITLE
feat(npm): gcp vpc flow logs doc

### PR DESCRIPTION
**Pull request by x8a**

### CODE REVIEW

**Comment by checonewrelic** [comment]
This looks good to me. @nr-jbiggley, please review.

**Comment by nr-jbiggley** [comment]
@x8a @checonewrelic Looks good to me for the initial launch. We may learn a bit from customers adopting post-launch that results in a cleanup of the language and flow, but let's get this into their hands sooner rather than later.

File: src/content/docs/network-performance-monitoring/setup-performance-monitoring/cloud-flow-logs/aws-vpc-flow-log-monitoring.mdx [hunk 1]
```diff
@@ -188,7 +188,7 @@ You can set up your Amazon VPC Flow Logs by installing a bundle that includes a
 Additionally, if you need to install more Amazon VPC Flow Logs or you don't want to install the bundle with a custom dashboard, follow these steps:
 
 1. Start the **[guided install process](https://one.newrelic.com/nr1-core?&state=bd947742-3034-63b7-7196-8baaf36dd8d9)**.
-2. From the **Select an account** dropdown, choose the account you want to send Amazon VPC Flow Logs to and click **Continue**.
+2. From the **Select an account** dropdown, choose the New Relic account you want to send Amazon VPC Flow Logs to and click **Continue**.
```
Comment (nbaenam) [inline src/content/docs/network-performance-monitoring/setup-performance-monitoring/cloud-flow-logs/aws-vpc-flow-log-monitoring.mdx:191]: I'd add here an oxford comma

File: src/content/docs/network-performance-monitoring/setup-performance-monitoring/cloud-flow-logs/gcp-vpc-flow-log-monitoring.mdx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+  - Network monitoring
+  - Installation
+  - Setup
+  - AWS
+metaDescription: Set up Amazon VPC Flow Logs monitoring.
+---
+
+import networkCloudFlowLogsOverview from 'images/network_screenshot-crop_cloud-flow-logs-overview.png'
+
+Google Cloud's Virtual Private Cloud Flow (VPC) Logs supports the frictionless transmission of logs to New Relic. With VPC flow logs from across your GCP estates, you can quickly understand key insights for performance analytics and troubleshooting network connectivity.
```
Comment (nbaenam) [inline src/content/docs/network-performance-monitoring/setup-performance-monitoring/cloud-flow-logs/gcp-vpc-flow-log-monitoring.mdx:14]: I'd add here an oxford comma

File: src/content/docs/network-performance-monitoring/setup-performance-monitoring/cloud-flow-logs/gcp-vpc-flow-log-monitoring.mdx [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    * The ability to create a Pub/Sub topic and set its permissions.
+    * The ability to create and update a Log Router.
+    * The ability to create a Dataflow job.
+
+## Send additional Google VPC Flow Logs [#additional-gcp-vpc-flow-logs]
+
+To send Google VPC flow logs to New Relic, follow these steps:
+
+1. Start the **[guided install process](https://one.newrelic.com/nr1-core?&state=bd947742-3034-63b7-7196-8baaf36dd8d9)**.
+2. From the **Select an account** dropdown, choose the New Relic account you want to send Google VPC Flow Logs to and click **Continue**.
```
Comment (nbaenam) [inline src/content/docs/network-performance-monitoring/setup-performance-monitoring/cloud-flow-logs/gcp-vpc-flow-log-monitoring.mdx:58]: I'd add here an oxford comma

**Review by nbaenam — APPROVED** [review]

**Pull request merged**